src/components/HeroSection.jsx

```
@@ -1,16 +1,121 @@
+        className="flex flex-col items-center justify-center px-4 pt-20 pb-20 mx-auto text-center sm:pt-52 md:pt-72 lg:pt-40"
+        variants={containerVariant}
+        initial="hidden"
+        whileInView="visible"
+        viewport={{ once: true }}
+      >
+        <motion.p
+          variants={itemsVariant}
+          viewport={{ once: true }}
+          className="flex items-center gap-2 px-6 py-1 mb-4 text-sm font-light border border-gray-200 rounded font-manrope"
+        >
+          You're welcome to
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)